Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,56 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
-        }
-    }
+
+
+			{
+				//Test for push function 
+				// test comment recording chnages test 
+				// test 2 
+				//CONTROL + F5
+				//Need to read Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm. 
+				string CarinaName = "Carina";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'CarinaName'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4844570)